internal/logic/assembler/transfer_event_validator.go

```diff
@@ -450,11 +460,34 @@ func (t *TransferEventMatcher) erc1155Matcher(transferEvents, gatewayEvents []ev
 				"gateway balance", info.GatewayBalance.String(),
 				"err info", info.Error,
 			)
-			slack.Notify(slack.MrkDwnGatewayTransferMessage(info))
-			return fmt.Errorf("erc1155 mismatch for token %s: gateway amount = %s, transfer amount = %s",
-				info.TokenAddress.Hex(), info.GatewayBalance.String(), info.TransferBalance.String())
+			return t.sendSlackAlert(info)
 		}
 	}
 
 	return nil
 }
+
+func (t *TransferEventMatcher) sendSlackAlert(info slack.GatewayTransferInfo) error {
+	info.TokenIgnored = t.isTokenIgnored(info.Layer, info.TokenAddress)
+	slack.Notify(slack.MrkDwnGatewayTransferMessage(info))
+	if info.TokenIgnored {
+		return nil
+	}
+	return fmt.Errorf("balance mismatch for token %s, token type = %s, transfer amount = %s, gateway amount = %s, info = %v",
```

Comment:
Yes. it will not be triggered immediately, but `chain-monitor`'s L1 (or L2) syncing will be stuck, and it will return invalid batch status, one on-call alarm will be triggered here.